prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -5,444 +5,21 @@ package gov.cdc.prime.router.cli
 import com.github.ajalt.clikt.completion.completionOption
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.core.subcommands
-import com.github.ajalt.clikt.parameters.groups.mutuallyExclusiveOptions
-import com.github.ajalt.clikt.parameters.groups.single
-import com.github.ajalt.clikt.parameters.options.convert
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.flag
 import com.github.ajalt.clikt.parameters.options.multiple
 import com.github.ajalt.clikt.parameters.options.option
-import com.github.ajalt.clikt.parameters.types.int
 import gov.cdc.prime.router.CsvComparer
-import gov.cdc.prime.router.DefaultValues
 import gov.cdc.prime.router.DocumentationFactory
-import gov.cdc.prime.router.FakeReport
 import gov.cdc.prime.router.FileSettings
-import gov.cdc.prime.router.FileSource
-import gov.cdc.prime.router.Hl7Configuration
 import gov.cdc.prime.router.Metadata
-import gov.cdc.prime.router.Report
-import gov.cdc.prime.router.ResultDetail
 import gov.cdc.prime.router.Schema
 import gov.cdc.prime.router.SettingsProvider
 import gov.cdc.prime.router.Translator
-import gov.cdc.prime.router.serializers.CsvSerializer
-import gov.cdc.prime.router.serializers.Hl7Serializer
-import gov.cdc.prime.router.serializers.RedoxSerializer
 import java.io.File
-import java.io.OutputStream
-import java.net.HttpURLConnection
-import java.net.URL
 import java.nio.file.Files
 import java.nio.file.Paths
```

Comment:
In the process of moving `ProcessData` I am able to strip a bunch of imports from main. I don't think this will have any practical performance benefit, but it will at least make the code a little cleaner.